tasks/86_working-hours/working-hours.js

```diff
@@ -0,0 +1,38 @@
+export function formatWorkingHours(arr) {
+  let start = null;
+  let end = null;
+  const tillDay = () => start !== end ? ` - ${end.day.toUpperCase()}` : '';
+  const dayAndHours = () => {
+    let startDay = start.day.toUpperCase();
+    return `${startDay}${tillDay()}: ${start.from} - ${start.to}`;
+  };
+  const daysOfWeek = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun'];
+  const sortedArray = arr.sort((a, b) => {
+    const indexOfEl = (day) => daysOfWeek.indexOf(day);
+    if (indexOfEl(a.day) < indexOfEl(b.day)) {
+      return -1;
+    } else {
+      return 1;
+    }
+  });
+
+  let scheduleArray = [];
+  for (let i = 0; i < sortedArray.length; i++) {
+    if (start === null) {
+      start = end = sortedArray[i];
+    } else if (sortedArray[i].from === start.from
+      && sortedArray[i].to === start.to) {
+      end = sortedArray[i];
+      if (!sortedArray[i + 1]) {
```

Comment:
and to not repeat yourself it better to have not in every if, but outside of them, or do this staff after loop, as anyway you need to do this